tests/unit/package_managers/python/uv/test_build_deps.py

```
@@ -0,0 +1,78 @@
+    package_dir = RootedPath(tmp_path)
+    output_dir = RootedPath(tmp_path / "output")
+    req_file = package_dir.join_within_root("requirements-build.txt")
+    req_file.path.write_text("setuptools==80.9.0\n")
+
+    result = download_build_dependencies(package_dir, output_dir)
+
+    assert result == mock_download.return_value
+    mock_download.assert_called_once_with(output_dir, [req_file], deps_dir_name="uv")
+
+
+def build_dep(package_type: str = "sdist", missing_checksum: bool = False) -> PyPIPackage:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)